src/SFA.DAS.EmployerAccounts/Models/Role.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace SFA.DAS.EmployerAccounts.Models
+{
+    public enum Role
```

Comment:
A minor point overall but this seemed like it could be improved, what do we think of this?

```
        private const string OwnerDescription = "Accept agreements, view information and manage PAYE schemes, organisations, apprentices and team members";
        private const string TransactorDescription = "Add apprentices and view information";
        private const string ViewerDescription = "View information but can’t make changes";


        public static string GetRoleDescription(this Role role, bool toLower = false)
        {
            string description;
            switch(role)
            {
                case Role.Owner: description = OwnerDescription; break;
                case Role.Transactor: description =TransactorDescription; break;
                case Role.Viewer: description = ViewerDescription; break;
                default: throw new ArgumentException($"Unexpected role: {nameof(role)}");
            }

return toLower ? string.Concat(char.ToLower(description[0]),description.Substring(1)) : description;
        }    
```

Which changed the calling line to this:
`Message = $"{email} can now {role.GetRoleDescription(true)}"`
instead of
`Message = $"{email} can now {RoleStrings.GetRoleDescriptionToLower(role)}"`

Also happy to remove the optional boolean and just always return the lower as that's all it needs to at the moment. 

Just a few thoughts.....

Reply:
I agree the code could be improved but I was trying to limit the scope of the changes (I didn't do very well did I? 😀) and this was existing code which has just been moved from somewhere else.